### DISCUSSION TITLE
Changing the location of navigation

### DISCUSSION THREAD

**Discussion by Salar-Sali** [nolimits4web/swiper#5996] [General]
Hello!
Is it possible to change the location of the navigation buttons? 
for example, outside of the swiper?
any help will be appreciated
Thanx in advance
![Screenshot from 2022-08-26 01-23-37](https://user-images.githubusercontent.com/100089076/186779410-7355f4a3-daa3-4cac-831a-3d5a291eff9e.png)

**Comment by Arhell** [comment]
Hi
example
https://codepen.io/bnmghjtyu/pen/djBMLR
this is for arrows, for navigation you can do it in the same way

**Reply by Salar-Sali** [reply]
Yeah it works this way.. thanx a lot

**Reply by Arhell** [reply]
Happy to help
